src/AppBundle/Entity/Report/ReportSubmission.php

```diff
@@ -69,7 +69,7 @@ class ReportSubmission
      * @ORM\OneToMany(targetEntity="AppBundle\Entity\Report\Document", mappedBy="reportSubmission")
      * @ORM\JoinColumn(name="report_submission_id", referencedColumnName="id", onDelete="CASCADE")
      *
-     * @ORM\OrderBy({"createdBy"="ASC"})
+     * @ORM\OrderBy({"createdOn"="ASC"})
```

Comment:
Need to check this change doesnt affect Documents download page in admin. There may be a reason why they're ordered by CreatedBy and possibly cause a bug to be raised due to changed behaviour.

Reply:
Good spot - the change was not required. Have reverted